spec/aws-record/record/batch_spec.rb

```diff
@@ -89,4 +92,177 @@
       end
     end
   end
+
+  describe '.read' do
+
+    let(:food) do
+      Class.new do
+        include(Aws::Record)
+        set_table_name('FoodTable')
+        integer_attr(:id, hash_key: true)
```

Comment:
I think we should test with a key that has a different database attribute name (to make sure that our logic correctly translates those in every place).